src/main/java/com/prgrms/domain/post/PostController.java

```diff
@@ -0,0 +1,59 @@
+package com.prgrms.domain.post;
+
+import com.prgrms.dto.PostDto;
+import com.prgrms.dto.PostDto.Response;
+import com.prgrms.dto.PostDto.ResponseArray;
+import java.net.URI;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("api/posts")
+public class PostController {
+
+    private final PostService service;
+
+    public PostController(@Validated PostService service) {
+        this.service = service;
+    }
+
+    @GetMapping("/{id}")
+    public ResponseEntity<PostDto.Response> getPostById(@PathVariable Long id) {
+
+        return ResponseEntity.ok(service.findPostById(id));
+    }
+
+    @PostMapping
+    public ResponseEntity<Void> registerPost(@RequestBody PostDto.Request postDto) {
+
+        Response responsePostDto = service.insertPost(postDto);
+
+        return ResponseEntity.created(URI.create("api/posts/" + responsePostDto.id())).build();
```

Comment:
코드 읽기가 어렵네요. 변수로 빼보면 좋을 거 같아요.
추가로 한줄에 한점만 찍을 수 있도록 변경해주세요
```suggestion
        return ResponseEntity.created(URI.create("api/posts/" + responsePostDto.id()))
        .build();
```